test/helpers/kubeobjects/environment/environment.go

```diff
@@ -3,28 +3,25 @@
 package environment
 
 import (
-	"os"
-
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/e2e-framework/klient/conf"
 	"sigs.k8s.io/e2e-framework/pkg/env"
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 	"sigs.k8s.io/e2e-framework/pkg/envfuncs"
+	"sigs.k8s.io/e2e-framework/support/kind"
 )
 
-const (
-	useKind = "TEST_ENV_USE_KIND"
-)
-
-func Get() env.Environment {
-	if os.Getenv(useKind) == "true" {
-		environment := env.New()
-		environment.Setup(envfuncs.CreateKindCluster(envconf.RandomName("operator-test", 10)))
-		return environment
-	}
+// let's decide here which config we really need to run on. use minimal required config.
+func CreateKindClusterEnvironment() env.Environment {
```

Comment:
Do we even need this?

Most things we deploy do not work on `Kind` AFAIK

Reply:
I agree with @0sewa0 we can create kind manually and set it to the current kube context. 
It would be nice in the future to add extra tags for tests we can run in kind or in GKE/ so it can be easy to run locally only for kind-specific tests.

Reply:
then I will remove the method (it's unused anyways) and we follow Andriis' suggestion in the future (separate ticket/pr)